commit 5a992a549adfe5a587bbcd6cd2b2cee47d236e27
Author: fanquake <fanquake@gmail.com>
Date:   Fri Sep 4 08:13:44 2020 +0800

    Work around broken mingw cross-compilation

    See upstream issues:
    https://bugreports.qt.io/browse/QTBUG-63637
    https://bugreports.qt.io/browse/QTBUG-63659
    https://codereview.qt-project.org/q/8bebded9

    We should be able to drop this once we are building qt 5.10.1 or later.

    Added in #12971.

diff --git a/qtbase/mkspecs/win32-g++/qmake.conf b/qtbase/mkspecs/win32-g++/qmake.conf
index e071a0d1..ad229b10 100644
--- a/qtbase/mkspecs/win32-g++/qmake.conf
+++ b/qtbase/mkspecs/win32-g++/qmake.conf
@@ -87,3 +87,5 @@ QMAKE_NM                = $${CROSS_COMPILE}nm -P
 include(../common/angle.conf)

 load(qt_config)
+QMAKE_LINK_OBJECT_MAX = 10
+QMAKE_LINK_OBJECT_SCRIPT = object_script
